menuinst/win32.py

```diff
@@ -251,26 +254,28 @@ def remove(self):
 
     def create(self, remove=False):
         args = []
+        # cmd is our root installation interpreter
+        cmd = join(unicode_prefix, u"pythonw.exe").replace("\\", "/")
+        # each of these roll the subprocess cmd into args
         if "pywscript" in self.shortcut:
-            cmd = join(self.menu.prefix, u"pythonw.exe").replace("\\", "/")
+            subprocess_cmd = join(self.menu.prefix, u"pythonw.exe").replace("\\", "/")
             args = self.shortcut["pywscript"].split()
-            args = get_python_args_for_subprocess(self.menu.prefix, args, cmd)
+            args = get_python_args_for_subprocess(self.menu.prefix, args, subprocess_cmd)
         elif "pyscript" in self.shortcut:
-            cmd = join(self.menu.prefix, u"python.exe").replace("\\", "/")
+            subprocess_cmd = join(self.menu.prefix, u"python.exe").replace("\\", "/")
```

Comment:
(omitting paths and file extensions for clarity)
Previously `cmd` was `"python"` (both for main and subprocess). Now `cmd` is `"pythonw"` (from above the `if`) and `subprocess_cmd` is `"python"`.
Thus the shortcuts change from
`python cwp python script`
to
`pythonw cwp python script`
. I don't know if we need to care about this at all... But maybe there is some special Windows voodoo for when to open up the terminal window or how to behave if the linked executable is a console or GUI application -- who knows